client/components/poll/closed-banner.js

```diff
@@ -18,18 +18,18 @@ const ClosedBanner = ( {
 		'crowdsignal-forms-poll__closed-banner'
 	);
 
+	let message = '';
+	if ( isPollHidden ) {
+		message = __( 'This Poll is Hidden', 'crowdsignal-forms' );
+	} else if ( !isPollHidden && isPollClosed ) {
+		message = __( 'This Poll is Closed', 'crowdsignal-forms' );
+	} else if ( !isPollHidden && hasVoted ) {
```

Comment:
```suggestion
	} else if ( isPollClosed ) {
		message = __( 'This Poll is Closed', 'crowdsignal-forms' );
	} else if ( hasVoted ) {
```

Reply:
As the first condition is isPollHidden, no need to replicate the condition for the else if

Reply:
Ahh, good call.  This has been amended.